src/assets/scripts/airport/AirportModel.js

```diff
@@ -39,9 +41,9 @@ const DEFAULT_CTR_CEILING_FT = 10000;
 const DEFAULT_INITIAL_ALTITUDE_FT = 5000;
```

Comment:
You are correct, some (though not all) _do_ belong in a more global context.  I have an issue outstanding for that very thing #49.

The thing with constants is that you want them to be only as global as they need to be.  Maybe they are needed for a specific function, or a class or even the entire app.  It's sometimes hard to tell what the scope should until you start using them.

But yes, there will be more and they will be global.